### PR TITLE
improve(affiliates): Run Dev mining rewards for week of 1/12

### LINKED ISSUE #2399: Run Dev mining rewards for week of 1/12
**Why**

EMP deployer rewards for 9th week from 2020-01-04 23:00:00 to 2021-01-11 23:00:00 utc

Contract | EMP Address | Payout Address
-- | -- | --
pxUSD_MAR2021 | 0x3a93E863cb3adc5910E6cea4d51f132E8666654F | 0x53911776641d6dF38B88b9eF27f920c617E3Cb5e
uGAS-JAN21 | 0x516f595978D87B67401DaB7AfD8555c3d28a3Af4 | 0x97990B693835da58A281636296D2Bf02787DEa17
Mario Cash-JAN-2021 | 0x4aa79c00240a2094ff3fa6cf7c67f521f32d84a2 | 0x7eb1e51351380E67F641B84545bf41eCdc10298E 
BCHNrBTC-JAN-2021 | 0xf32219331a03d99c98adf96d43cc312353003531 | 0x37a0b32e07CD0b53AFb2981Cf02A74e10B80B577 
YD-BTC-MAR21 | 0x1c3f1A342c8D9591D9759220d114C685FD1cF6b8 | 0xa0dfF1F09a0DCEda7641b7eB07c3BDbe66a06C35
YD-ETH-MAR21 | 0xE4256C47a3b27a969F25de8BEf44eCA5F2552bD5 | 0xa0dfF1F09a0DCEda7641b7eB07c3BDbe66a06C35
Zelda "Spring Nuts" USDC | 0xeAddB6AD65dcA45aC3bB32f88324897270DA0387  | 0xF53FBC3aDbe91E1A408716C0AF4C0cDC17A2cca8
Zelda "Summer Nuts" USDC | 0xf215778f3a5e7ab6a832e71d87267dd9a9ab0037 | 0xb8E237CC8D2E81b7C9C9C4B5877200f85100fC87
Zelda "Elastic" USDC |0x267D46e71764ABaa5a0dD45260f95D9c8d5b8195 | 0x531B290230b1809d0AE29Ea834EdF96748F49Bf9
Zelda.Cash "White Sword" WETH |0x45c4dbd73294c5d8ddf6e5f949be4c505e6e9495 |0xa1eD088d0789e7baBa18Ed4d87baca65376850d8
Zelda "Elastic" USDC | 0xda0943251079eb9f517668fdb372fc6ae299d898 | 0xdB0d599C3D49F6B349c83Dc2407a16F2830d6fCc
Zelda "Spring Shield" renBTC |0x267D46e71764ABaa5a0dD45260f95D9c8d5b8195 | 0x531B290230b1809d0AE29Ea834EdF96748F49Bf9
Zelda "Summer Shield" renBTC |0xd6fc1a7327210b7fe33ef2514b44979719424a1d | 0xc107a610559bF7c7853e676333c53AF135E89a39
Zelda Whirlwind Cash |0x2862a798b3defc1c24b9c0d241beaf044c45e585 | 0xa1eD088d0789e7baBa18Ed4d87baca65376850d8
Zelda Reinforced Cash |0xd81028a6fbaaaf604316f330b20d24bfbfd14478 | 0xdB0d599C3D49F6B349c83Dc2407a16F2830d6fCc

### PR DESCRIPTION
**Pull request by daywiss**
**Motivation**

#2399 

**Summary**

Adds dev mining rewards and updates dev mining status file to reflect currently whitelisted emps.

This weeks rewards ran into issues, and execution is dependent on a currently pending PR #2398 .  In order to support EMPs which are dollar denominated but do not have active price feeds, we add a fallback price to 1$ for several emps which you can see denoted as `fallbackPrices` in the config.

**Details**

To run create a config file called week9.json with this info:
```
{
    "empWhitelist": [
      ["0x3a93E863cb3adc5910E6cea4d51f132E8666654F", "0x53911776641d6dF38B88b9eF27f920c617E3Cb5e"],
      ["0x516f595978D87B67401DaB7AfD8555c3d28a3Af4", "0x97990B693835da58A281636296D2Bf02787DEa17"],
      ["0x4AA79c00240a2094Ff3fa6CF7c67f521f32D84a2", "0x7eb1e51351380E67F641B84545bf41eCdc10298E"],
      ["0xf32219331A03D99C98Adf96D43cc312353003531", "0x37a0b32e07CD0b53AFb2981Cf02A74e10B80B577"],
      ["0x1c3f1A342c8D9591D9759220d114C685FD1cF6b8", "0xa0dfF1F09a0DCEda7641b7eB07c3BDbe66a06C35"],
      ["0xE4256C47a3b27a969F25de8BEf44eCA5F2552bD5", "0xa0dfF1F09a0DCEda7641b7eB07c3BDbe66a06C35"],
      ["0xeAddB6AD65dcA45aC3bB32f88324897270DA0387", "0xF53FBC3aDbe91E1A408716C0AF4C0cDC17A2cca8"],
      ["0xf215778f3a5e7ab6a832e71d87267dd9a9ab0037", "0xb8E237CC8D2E81b7C9C9C4B5877200f85100fC87"],
      ["0x267D46e71764ABaa5a0dD45260f95D9c8d5b8195", "0x531B290230b1809d0AE29Ea834EdF96748F49Bf9"],
      ["0x45c4dbd73294c5d8ddf6e5f949be4c505e6e9495", "0xa1eD088d0789e7baBa18Ed4d87baca65376850d8"],
      ["0xda0943251079eb9f517668fdb372fc6ae299d898", "0xdB0d599C3D49F6B349c83Dc2407a16F2830d6fCc"],
      ["0xd6fc1a7327210b7fe33ef2514b44979719424a1d", "0xc107a610559bF7c7853e676333c53AF135E89a39"],
      ["0x2862a798b3defc1c24b9c0d241beaf044c45e585", "0xa1eD088d0789e7baBa18Ed4d87baca65376850d8"],
      ["0xd81028a6fbaaaf604316f330b20d24bfbfd14478", "0xdB0d599C3D49F6B349c83Dc2407a16F2830d6fCc"]
    ],
    "startTime": 1609801200000,
    "endTime": 1610406000000,
    "totalRewards": "50000",
    "network": 1,
    "fallbackPrices": [
      ["0xeAddB6AD65dcA45aC3bB32f88324897270DA0387", "1"],
      ["0xf215778f3a5e7ab6a832e71d87267dd9a9ab0037", "1"],
      ["0x267D46e71764ABaa5a0dD45260f95D9c8d5b8195", "1"],
      ["0x45c4dbd73294c5d8ddf6e5f949be4c505e6e9495", "1"],
      ["0xda0943251079eb9f517668fdb372fc6ae299d898", "1"],
      ["0xd6fc1a7327210b7fe33ef2514b44979719424a1d", "1"],
      ["0x2862a798b3defc1c24b9c0d241beaf044c45e585", "1"],
      ["0xd81028a6fbaaaf604316f330b20d24bfbfd14478", "1"]
    ]
}
```
and in terminal inside affiliates package run:

node apps/DevMiningRewards week9.json --network=mainnet_mnemonic

**Issue(s)**

Fixes #2399

### CODE REVIEW

**Review by chrismaree — APPROVED** [review]
looks great! love how many new projects there are added each week.

File: packages/affiliates/payouts/devmining/2021-01-04_2021-01-11.json [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    "empWhitelist": [
+      ["0x3a93E863cb3adc5910E6cea4d51f132E8666654F", "0x53911776641d6dF38B88b9eF27f920c617E3Cb5e"],
+      ["0x516f595978D87B67401DaB7AfD8555c3d28a3Af4", "0x97990B693835da58A281636296D2Bf02787DEa17"],
+      ["0x4AA79c00240a2094Ff3fa6CF7c67f521f32D84a2", "0x7eb1e51351380E67F641B84545bf41eCdc10298E"],
+      ["0xf32219331A03D99C98Adf96D43cc312353003531", "0x37a0b32e07CD0b53AFb2981Cf02A74e10B80B577"],
+      ["0x1c3f1A342c8D9591D9759220d114C685FD1cF6b8", "0xa0dfF1F09a0DCEda7641b7eB07c3BDbe66a06C35"],
+      ["0xE4256C47a3b27a969F25de8BEf44eCA5F2552bD5", "0xa0dfF1F09a0DCEda7641b7eB07c3BDbe66a06C35"],
+      ["0xeAddB6AD65dcA45aC3bB32f88324897270DA0387", "0xF53FBC3aDbe91E1A408716C0AF4C0cDC17A2cca8"],
+      ["0xf215778f3a5e7ab6a832e71d87267dd9a9ab0037", "0xb8E237CC8D2E81b7C9C9C4B5877200f85100fC87"],
+      ["0x267D46e71764ABaa5a0dD45260f95D9c8d5b8195", "0x531B290230b1809d0AE29Ea834EdF96748F49Bf9"],
```
Comment (smb2796) [inline packages/affiliates/payouts/devmining/2021-01-04_2021-01-11.json:12]: This EMP is counted twice in the whitelist and fallbackPrices. Not sure if it affected the rewards amounts though

**Review by smb2796 — COMMENTED** [review]
Payout addresses and reward proportionality look correct. Only edit is the comment I put about listing `0x267D46e71764ABaa5a0dD45260f95D9c8d5b8195` twice.

File: packages/affiliates/payouts/devmining/2021-01-04_2021-01-11.json (hunk 1 above)
Comment (daywiss) [reply packages/affiliates/payouts/devmining/2021-01-04_2021-01-11.json:12]: thanks good catch.

**Review by smb2796 — APPROVED** [review]
LGTM!

**Comment by hlambur** [comment]
These rewards have been sent!

**Pull request merged**